shared/src/commonMain/kotlin/xyz/alaniz/aaron/lightsaber/ui/settings/SettingsUi.kt

```diff
@@ -192,12 +218,40 @@ fun LightsaberBladeColorDropdownMenu(
 }
 
 @Composable
-private fun LightsaberCircle(color: Color, modifier: Modifier = Modifier) {
+private fun LightsaberCircle(
+    color: Color,
+    modifier: Modifier = Modifier,
+    customColor: Color? = null
+) {
```

Comment:
Is there we this can be refactored to just pass in color instead of both color and customColor? I also noticed there is a bug where after you select a custom color, the drop down menu circle does not match the selected color.